content/client-server-api/_index.md

```diff
@@ -1485,6 +1489,72 @@ and other APIs to detect when the lock has been lifted.
 To enable users to appeal to a lock clients MAY use
 [server contact discovery](#getwell-knownmatrixsupport).
 
+#### Account suspension
+
+{{% added-in v="1.13" %}}
+
+Similar to [locking](#account-locking), a user's account MAY be suspended to
+prevent further activity from that account. A similar effect is achieved, though
```

Comment:
I found the "A similar effect is achieved" part hard to grok. Also, avoid passive voice.

```suggestion
Server administrators MAY suspend a user's account to prevent further activity
from that account. The effect is similar to [locking](#account-locking), though
```